### PR TITLE
v5.0.0-beta.2

### PR DESCRIPTION
**Pull request by m4theushw**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,60 @@
...
+  +  /**
+  +   * The cell value, but if the column has valueGetter, use getValue.
+  +   */
+  +  value: GridCellValue;
+  +  /**
+  +   * GridApi that let you manipulate the grid.
+  +   */
+  +  api: any;
+  +}
+  ```
```
Comment (oliviertassinari) [inline CHANGELOG.md:40]: Would it make sense to revert this breaking change? It's a beta release, we have exited the alpha state, it didn't seem needed to fix #2514.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,60 @@
...
 
+## 5.0.0-beta.2
+
+_Sep 23, 2021_
+
+A big thanks to the 5 contributors who made this release possible. Here are some highlights ✨:
+
+- 🇻🇳 Add Vietnamese (viVN) locale (#2668) @tuananh281098
+- 🇵🇱 Improve Polish (plPL) locale (#2632) @michallukowski
+- ⚡️ Apply the `valueFormatter` to `singleSelect` select options (#2581) @DanailH
```
Comment (oliviertassinari) [inline CHANGELOG.md:14]: It's a bug fix, right?

```suggestion
- 🐛 Fix usage of `valueFormatter` with the `singleSelect` column type (#2581) @DanailH
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,60 @@
...
+
+### `@mui/x-data-grid@v5.0.0-beta.2` / `@mui/x-data-grid-pro@v5.0.0-beta.2`
+
+#### Breaking changes
+
+- [DataGrid] The params passed to the `valueFormatter` were changed. (#2581) @DanailH
+  Use the `api` to get the missing params.
```
Comment (oliviertassinari) [inline CHANGELOG.md:21]: To fix the formatting: https://github.com/mui-org/material-ui-x/blob/65a0671b2983a63d6107cebf7e0cb02093a05ed9/CHANGELOG.md

```suggestion

  Use the `api` to get the missing params.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -11,13 +65,16 @@ A big thanks to the 3 contributors who made this release possible.
 
 ### `@mui/x-data-grid@v5.0.0-beta.1` / `@mui/x-data-grid-pro@v5.0.0-beta.1`
 
-#### Docs
+There are no changes since the last release.
+Refer to `v5.0.0-beta.0` for the breaking changes.
```
Comment (oliviertassinari) [inline CHANGELOG.md:69]: Would it be clearer?

```suggestion
This is a hot fix, to fix an important regression with 5.0.0-beta.0.
```

File: CHANGELOG.md (hunk 2 above)
Comment (flaviendelangle) [reply CHANGELOG.md:14]: It was not applied at all before

Comment (m4theushw) [reply CHANGELOG.md:14]: It's more of a feature.

Comment (m4theushw) [reply CHANGELOG.md:14]:
```suggestion
- ⚡️ Apply the `valueFormatter` to the `singleSelect` column type (#2581) @DanailH
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,60 @@
...
+- [DataGrid] Add Vietnamese (viVN) locale (#2668) @tuananh281098
+- [DataGrid] Apply the `valueFormatter` to `singleSelect` select options (#2581) @DanailH
+- [DataGrid] Free up column header space when icons are not visible (#2606) @DanailH
+- [DataGrid] Improve Polish (plPL) locale (#2632) @michallukowski
+
+### Docs
+
+- [docs] Add section for controlled selection and server-side pagination (#2602) @DanailH
+- [docs] Fix Algolia search (#2655) @oliviertassinari
+- [docs] Fix the seach results to make them helpful (#2656) @oliviertassinari
```
Comment (flaviendelangle) [inline CHANGELOG.md:53]:
```suggestion
- [docs] Improve the seach results relevance (#2656) @oliviertassinari
```

**Review by oliviertassinari — APPROVED** [review]

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,61 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.0.0-beta.2
+
+_Sep 23, 2021_
```
Comment (m4theushw) [inline CHANGELOG.md:8]:
```suggestion
_Sep 24, 2021_
```

**Pull request merged**